---
title: Identification of Evolutionary Trajectories Shared across Human Betacoronaviruses
description:
url: ''
date: '2023-06-01'
publishDate: '2025-03-07T18:12:05.423416Z'
authors:
- Marina Escalera-Zamudio
- Sergei L Kosakovsky Pond
- Natalia Martı́nez de la Viña
- Bernardo Gutiérrez
- Rhys P D Inward
- Julien Thézé
- Lucy van Dorp
- Hugo G Castelán-Sánchez
- Thomas A Bowden
- Oliver G Pybus
- Ruben J G Hulswit
publication_types:
- '2'
abstract: Comparing the evolution of distantly related viruses can provide insights
  into common adaptive processes related to shared ecological niches. Phylogenetic
  approaches, coupled with other molecular evolution tools, can help identify mutations
  informative on adaptation, although the structural contextualization of these to
  functional sites of proteins may help gain insight into their biological properties.
  Two zoonotic betacoronaviruses capable of sustained human-to-human transmission
  have caused pandemics in recent times (SARS-CoV-1 and SARS-CoV-2), although a third
  virus (MERS-CoV) is responsible for sporadic outbreaks linked to animal infections.
  Moreover, two other betacoronaviruses have circulated endemically in humans for
  decades (HKU1 and OC43). To search for evidence of adaptive convergence between
  established and emerging betacoronaviruses capable of sustained human-to-human transmission
  (HKU1, OC43, SARS-CoV-1, and SARS-CoV-2), we developed a methodological pipeline
  to classify shared nonsynonymous mutations as putatively denoting homoplasy (repeated
  mutations that do not share direct common ancestry) or stepwise evolution (sequential
  mutations leading towards a novel genotype). In parallel, we look for evidence of
  positive selection and draw upon protein structure data to identify potential biological
  implications. We find 30 candidate mutations, from which 4 (codon sites 18121 [nsp14/residue
  28], 21623 [spike/21], 21635 [spike/25], and 23948 [spike/796]; SARS-CoV-2 genome
  numbering) further display evolution under positive selection and proximity to functional
  protein regions. Our findings shed light on potential mechanisms underlying betacoronavirus
  adaptation to the human host and pinpoint common mutational pathways that may occur
  during establishment of human endemicity.
featured: false
publication: '*Genome Biol Evol*'
tags:
- adaptation; betacoronaviruses; convergence; molecular evolution; phylogenomics
doi: 10.1093/gbe/evad076
---

Page content here