---
title: Comprehensive sieve analysis of breakthrough HIV-1 sequences in the RV144 vaccine
  efficacy trial
description:
url: ''
date: '2015-02-01'
publishDate: '2025-03-07T18:12:13.211755Z'
authors:
- Paul T Edlefsen
- Morgane Rolland
- Tomer Hertz
- Sodsai Tovanabutra
- Andrew J Gartland
- Allan C deCamp
- Craig A Magaret
- Hasan Ahmed
- Raphael Gottardo
- Michal Juraska
- Connor McCoy
- Brendan B Larsen
- Eric Sanders-Buell
- Chris Carrico
- Sergey Menis
- Meera Bose
- RV144 Sequencing Team
- Miguel A Arroyo
- Robert J O'Connell
- Sorachai Nitayaphan
- Punnee Pitisuttithum
- Jaranit Kaewkungwal
- Supachai Rerks-Ngarm
- Merlin L Robb
- Tatsiana Kirys
- Ivelin S Georgiev
- Peter D Kwong
- Konrad Scheffler
- Sergei L Kosakovsky Pond
- Jonathan M Carlson
- Nelson L Michael
- William R Schief
- James I Mullins
- Jerome H Kim
- Peter B Gilbert
publication_types:
- '2'
abstract: The RV144 clinical trial showed the partial efficacy of a vaccine regimen
  with an estimated vaccine efficacy (VE) of 31% for protecting low-risk Thai volunteers
  against acquisition of HIV-1. The impact of vaccine-induced immune responses can
  be investigated through sieve analysis of HIV-1 breakthrough infections (infected
  vaccine and placebo recipients). A V1/V2-targeted comparison of the genomes of HIV-1
  breakthrough viruses identified two V2 amino acid sites that differed between the
  vaccine and placebo groups. Here we extended the V1/V2 analysis to the entire HIV-1
  genome using an array of methods based on individual sites, k-mers and genes/proteins.
  We identified 56 amino acid sites or \"signatures\" and 119 k-mers that differed
  between the vaccine and placebo groups. Of those, 19 sites and 38 k-mers were located
  in the regions comprising the RV144 vaccine (Env-gp120, Gag, and Pro). The nine
  signature sites in Env-gp120 were significantly enriched for known antibody-associated
  sites (p = 0.0021). In particular, site 317 in the third variable loop (V3) overlapped
  with a hotspot of antibody recognition, and sites 369 and 424 were linked to CD4
  binding site neutralization. The identified signature sites significantly covaried
  with other sites across the genome (mean = 32.1) more than did non-signature sites
  (mean = 0.9) (p < 0.0001), suggesting functional and/or structural relevance of
  the signature sites. Since signature sites were not preferentially restricted to
  the vaccine immunogens and because most of the associations were insignificant following
  correction for multiple testing, we predict that few of the genetic differences
  are strongly linked to the RV144 vaccine-induced immune pressure. In addition to
  presenting results of the first complete-genome analysis of the breakthrough infections
  in the RV144 trial, this work describes a set of statistical methods and tools applicable
  to analysis of breakthrough infection genomes in general vaccine efficacy trials
  for diverse pathogens.
featured: false
publication: '*PLoS Comput Biol*'
doi: 10.1371/journal.pcbi.1003973
---

Page content here