---
title: 'IDEPI: rapid prediction of HIV-1 antibody epitopes and other phenotypic features
  from sequence data using a flexible machine learning platform'
description:
url: ''
date: '2014-09-01'
publishDate: '2025-03-07T18:12:13.631035Z'
authors:
- N Lance Hepler
- Konrad Scheffler
- Steven Weaver
- Ben Murrell
- Douglas D Richman
- Dennis R Burton
- Pascal Poignard
- Davey M Smith
- Sergei L Kosakovsky Pond
publication_types:
- '2'
abstract: 'Since its identification in 1983, HIV-1 has been the focus of a research
  effort unprecedented in scope and difficulty, whose ultimate goals--a cure and a
  vaccine--remain elusive. One of the fundamental challenges in accomplishing these
  goals is the tremendous genetic variability of the virus, with some genes differing
  at as many as 40% of nucleotide positions among circulating strains. Because of
  this, the genetic bases of many viral phenotypes, most notably the susceptibility
  to neutralization by a particular antibody, are difficult to identify computationally.
  Drawing upon open-source general-purpose machine learning algorithms and libraries,
  we have developed a software package IDEPI (IDentify EPItopes) for learning genotype-to-phenotype
  predictive models from sequences with known phenotypes. IDEPI can apply learned
  models to classify sequences of unknown phenotypes, and also identify specific sequence
  features which contribute to a particular phenotype. We demonstrate that IDEPI achieves
  performance similar to or better than that of previously published approaches on
  four well-studied problems: finding the epitopes of broadly neutralizing antibodies
  (bNab), determining coreceptor tropism of the virus, identifying compartment-specific
  genetic signatures of the virus, and deducing drug-resistance associated mutations.
  The cross-platform Python source code (released under the GPL 3.0 license), documentation,
  issue tracking, and a pre-configured virtual machine for IDEPI can be found at https://github.com/veg/idepi.'
featured: false
publication: '*PLoS Comput Biol*'
doi: 10.1371/journal.pcbi.1003842
---

Page content here