---
title: Single-Cell Transcriptomics Reveal a Correlation between Genome Architecture
  and Gene Family Evolution in Ciliates
description:
url: ''
date: '2019-12-01'
publishDate: '2025-03-07T18:12:09.428211Z'
authors:
- Ying Yan
- Xyrus X Maurer-Alcal√°
- Rob Knight
- Sergei L Kosakovsky Pond
- Laura A Katz
publication_types:
- '2'
abstract: 'Ciliates, a eukaryotic clade that is over 1 billion years old, are defined
  by division of genome function between transcriptionally inactive germline micronuclei
  and functional somatic macronuclei. To date, most analyses of gene family evolution
  have been limited to cultivable model lineages (e.g., Tetrahymena, Paramecium, Oxytricha,
  and Stylonychia). Here, we focus on the uncultivable Karyorelictea and its understudied
  sister class Heterotrichea, which represent two extremes in genome architecture.
  Somatic macronuclei within the Karyorelictea are described as nearly diploid, while
  the Heterotrichea have hyperpolyploid somatic genomes. Previous analyses indicate
  that genome architecture impacts ciliate gene family evolution as the most diverse
  and largest gene families are found in lineages with extensively processed somatic
  genomes (i.e., possessing thousands of gene-sized chromosomes). To further assess
  ciliate gene family evolution, we analyzed 43 single-cell transcriptomes from 33
  ciliate species representing 10 classes. Focusing on conserved eukaryotic genes,
  we use estimates of transcript diversity as a proxy for the number of paralogs in
  gene families among four focal clades: Karyorelictea, Heterotrichea, extensive fragmenters
  (with gene-size somatic chromosomes), and non-extensive fragmenters (with more traditional
  somatic chromosomes), the latter two within the subphylum Intramacronucleata. Our
  results show that (i) the Karyorelictea have the lowest average transcript diversity,
  while Heterotrichea are highest among the four groups; (ii) proteins in Karyorelictea
  are under the highest functional constraints, and the patterns of selection in ciliates
  may reflect genome architecture; and (iii) stop codon reassignments vary among members
  of the Heterotrichea and Spirotrichea but are conserved in other classes.IMPORTANCE
  To further our understanding of genome evolution in eukaryotes, we assess the relationship
  between patterns of molecular evolution within gene families and variable genome
  structures found among ciliates. We combine single-cell transcriptomics with bioinformatic
  tools, focusing on understudied and uncultivable lineages selected from across the
  ciliate tree of life. Our analyses show that genome architecture correlates with
  patterns of protein evolution as lineages with more canonical somatic genomes, such
  as the class Karyorelictea, have more conserved patterns of molecular evolution
  compared to other classes. This study showcases the power of single-cell transcriptomics
  for investigating genome architecture and evolution in uncultivable microbial lineages
  and provides transcriptomic resources for further research on genome evolution.'
featured: false
publication: '*mBio*'
tags:
- Ciliophora; gene family evolution; genetic code evolution; phylogenomics; transcriptomics;
  uncultivable microbes
doi: 10.1128/mBio.02524-19
---

Page content here