---
title: HIV-specific probabilistic models of protein evolution
description:
url: ''
date: '2007-01-01'
publishDate: '2025-03-07T18:12:19.647832Z'
authors:
- David C Nickle
- Laura Heath
- Mark A Jensen
- Peter B Gilbert
- James I Mullins
- Sergei L Kosakovsky Pond
publication_types:
- '2'
abstract: Comparative sequence analyses, including such fundamental bioinformatics
  techniques as similarity searching, sequence alignment and phylogenetic inference,
  have become a mainstay for researchers studying type 1 Human Immunodeficiency Virus
  (HIV-1) genome structure and evolution. Implicit in comparative analyses is an underlying
  model of evolution, and the chosen model can significantly affect the results. In
  general, evolutionary models describe the probabilities of replacing one amino acid
  character with another over a period of time. Most widely used evolutionary models
  for protein sequences have been derived from curated alignments of hundreds of proteins,
  usually based on mammalian genomes. It is unclear to what extent these empirical
  models are generalizable to a very different organism, such as HIV-1-the most extensively
  sequenced organism in existence. We developed a maximum likelihood model fitting
  procedure to a collection of HIV-1 alignments sampled from different viral genes,
  and inferred two empirical substitution models, suitable for describing between-and
  within-host evolution. Our procedure pools the information from multiple sequence
  alignments, and provided software implementation can be run efficiently in parallel
  on a computer cluster. We describe how the inferred substitution models can be used
  to generate scoring matrices suitable for alignment and similarity searches. Our
  models had a consistently superior fit relative to the best existing models and
  to parameter-rich data-driven models when benchmarked on independent HIV-1 alignments,
  demonstrating evolutionary biases in amino-acid substitution that are unique to
  HIV, and that are not captured by the existing models. The scoring matrices derived
  from the models showed a marked difference from common amino-acid scoring matrices.
  The use of an appropriate evolutionary model recovered a known viral transmission
  history, whereas a poorly chosen model introduced phylogenetic error. We argue that
  our model derivation procedure is immediately applicable to other organisms with
  extensive sequence data available, such as Hepatitis C and Influenza A viruses.
featured: false
publication: '*PLoS One*'
doi: 10.1371/journal.pone.0000503
---

Page content here