---
title: Computational analysis and identification of amino acid sites in dengue E proteins
  relevant to development of diagnostics and vaccines
description:
url: ''
date: '2007-10-01'
publishDate: '2025-03-07T18:12:19.920767Z'
authors:
- Raja Mazumder
- Zhang-Zhi Hu
- C R Vinayaka
- Jose-Luis Sagripanti
- Simon D W Frost
- Sergei L Kosakovsky Pond
- Cathy H Wu
publication_types:
- '2'
abstract: We have identified 72 completely conserved amino acid residues in the E
  protein of major groups of the Flavivirus genus by computational analyses. In the
  dengue species we have identified 12 highly conserved sequence regions, 186 negatively
  selected sites, and many dengue serotype-specific negatively selected sites. The
  flavivirus-conserved sites included residues involved in forming six disulfide bonds
  crucial for the structural integrity of the protein, the fusion motif involved in
  viral infectivity, and the interface residues of the oligomers. The structural analysis
  of the E protein showed 19 surface-exposed non-conserved residues, 128 dimer or
  trimer interface residues, and regions, which undergo major conformational change
  during trimerization. Eleven consensus T(h)-cell epitopes common to all four dengue
  serotypes were predicted. Most of these corresponded to dengue-conserved regions
  or negatively selected sites. Of special interest are six singular sites (N(37),
  Q(211), D(215), P(217), H(244), K(246)) in dengue E protein that are conserved,
  are part of the predicted consensus T(h)-cell epitopes and are exposed in the dimer
  or trimer. We propose these sites and corresponding epitopic regions as potential
  candidates for prioritization by experimental biologists for development of diagnostics
  and vaccines that may be difficult to circumvent by natural or man-made alteration
  of dengue virus.
featured: false
publication: '*Virus Genes*'
doi: 10.1007/s11262-007-0103-2
---

Page content here