---
title: Evolutionary fingerprinting of genes
description:
url: ''
date: '2010-03-01'
publishDate: '2025-03-07T18:12:17.846757Z'
authors:
- Sergei L Kosakovsky Pond
- Konrad Scheffler
- Michael B Gravenor
- Art F Y Poon
- Simon D W Frost
publication_types:
- '2'
abstract: Over time, natural selection molds every gene into a unique mosaic of sites
  evolving rapidly or resisting change-an \"evolutionary fingerprint\" of the gene.
  Aspects of this evolutionary fingerprint, such as the site-specific ratio of nonsynonymous
  to synonymous substitution rates (dN/dS), are commonly used to identify genetic
  features of potential biological interest; however, no framework exists for comparing
  evolutionary fingerprints between genes. We hypothesize that protein-coding genes
  with similar protein structure and/or function tend to have similar evolutionary
  fingerprints and that comparing evolutionary fingerprints can be useful for discovering
  similarities between genes in a way that is analogous to, but independent of, discovery
  of similarity via sequence-based comparison tools such as Blast. To test this hypothesis,
  we develop a novel model of coding sequence evolution that uses a general bivariate
  discrete parameterization of the evolutionary rates. We show that this approach
  provides a better fit to the data using a smaller number of parameters than existing
  models. Next, we use the model to represent evolutionary fingerprints as probability
  distributions and present a methodology for comparing these distributions in a way
  that is robust against variations in data set size and divergence. Finally, using
  sequences of three rapidly evolving RNA viruses (HIV-1, hepatitis C virus, and influenza
  A virus), we demonstrate that genes within the same functional group tend to have
  similar evolutionary fingerprints. Our framework provides a sound statistical foundation
  for efficient inference and comparison of evolutionary rate patterns in arbitrary
  collections of gene alignments, clustering homologous and nonhomologous genes, and
  investigation of biological and functional correlates of evolutionary rates.
featured: false
publication: '*Mol Biol Evol*'
doi: 10.1093/molbev/msp260
---

Page content here