---
title: 'Evolution of Viral Genomes: Interplay Between Selection, Recombination, and
  Other Forces'
description:
url: ''
date: '2019-01-01'
publishDate: '2025-03-07T18:12:09.715897Z'
authors:
- Stephanie J Spielman
- Steven Weaver
- Stephen D Shank
- Brittany Rife Magalis
- Michael Li
- Sergei L Kosakovsky Pond
publication_types:
- '2'
abstract: Natural selection is a fundamental force shaping organismal evolution, as
  it both maintains function and enables adaptation and innovation. Viruses, with
  their typically short and largely coding genomes, experience strong and diverse
  selective forces, sometimes acting on timescales that can be directly measured.
  These selection pressures emerge from an antagonistic interplay between rapidly
  changing fitness requirements (immune and antiviral responses from hosts, transmission
  between hosts, or colonization of new host species) and functional imperatives (the
  ability to infect hosts or host cells and replicate within hosts). Indeed, computational
  methods to quantify these evolutionary forces using molecular sequence data were
  initially, dating back to the 1980s, applied to the study of viral pathogens. This
  preference largely emerged because the strong selective forces are easiest to detect
  in viruses, and, of course, viruses have clear biomedical relevance. Recent commoditization
  of affordable high-throughput sequencing has made it possible to generate truly
  massive genomic data sets, on which powerful and accurate methods can yield a very
  detailed depiction of when, where, and (sometimes) how viral pathogens respond to
  various selective forces.Here, we present recent statistical developments and state-of-the-art
  methods to identify and characterize these selection pressures from protein-coding
  sequence alignments and phylogenies. Methods described here can reveal critical
  information about various evolutionary regimes, including whole-gene selection,
  lineage-specific selection, and site-specific selection acting upon viral genomes,
  while accounting for confounding biological processes, such as recombination and
  variation in mutation rates.
featured: false
publication: '*Methods Mol Biol*'
tags:
- Codon models; Molecular evolution; Phylogenetics; Positive selection; Recombination;
  Relaxed selection; Virus evolution
doi: 10.1007/978-1-4939-9074-0_14
---

Page content here