---
title: Correcting the bias of empirical frequency parameter estimators in codon models
description:
url: ''
date: '2010-01-01'
publishDate: '2025-03-07T18:12:17.058364Z'
authors:
- Sergei L Kosakovsky Pond
- Wayne Delport
- Spencer V Muse
- Konrad Scheffler
publication_types:
- '2'
abstract: Markov models of codon substitution are powerful inferential tools for studying
  biological processes such as natural selection and preferences in amino acid substitution.
  The equilibrium character distributions of these models are almost always estimated
  using nucleotide frequencies observed in a sequence alignment, primarily as a matter
  of historical convention. In this note, we demonstrate that a popular class of such
  estimators are biased, and that this bias has an adverse effect on goodness of fit
  and estimates of substitution rates. We propose a \"corrected\" empirical estimator
  that begins with observed nucleotide counts, but accounts for the nucleotide composition
  of stop codons. We show via simulation that the corrected estimates outperform the
  de facto standard estimates not just by providing better estimates of the frequencies
  themselves, but also by leading to improved estimation of other parameters in the
  evolutionary models. On a curated collection of sequence alignments, our estimators
  show a significant improvement in goodness of fit compared to the approach. Maximum
  likelihood estimation of the frequency parameters appears to be warranted in many
  cases, albeit at a greater computational cost. Our results demonstrate that there
  is little justification, either statistical or computational, for continued use
  of the -style estimators.
featured: false
publication: '*PLoS One*'
doi: 10.1371/journal.pone.0011230
---

Page content here