---
title: A maximum likelihood method for detecting directional evolution in protein
  sequences and its application to influenza A virus
description:
url: ''
date: '2008-09-01'
publishDate: '2025-03-07T18:12:18.926724Z'
authors:
- Sergei L Kosakovsky Pond
- Art F Y Poon
- Andrew J Leigh Brown
- Simon D W Frost
publication_types:
- '2'
abstract: We develop a model-based phylogenetic maximum likelihood test for evidence
  of preferential substitution toward a given residue at individual positions of a
  protein alignment--directional evolution of protein sequences (DEPS). DEPS can identify
  both the target residue and sites evolving toward it, help detect selective sweeps
  and frequency-dependent selection--scenarios that confound most existing tests for
  selection, and achieve good power and accuracy on simulated data. We applied DEPS
  to alignments representing different genomic regions of influenza A virus (IAV),
  sampled from avian hosts (H5N1 serotype) and human hosts (H3N2 serotype), and identified
  multiple directionally evolving sites in 5/8 genomic segments of H5N1 and H3N2 IAV.
  We propose a simple descriptive classification of directionally evolving sites into
  5 groups based on the temporal distribution of residue frequencies and document
  known functional correlates, such as immune escape or host adaptation.
featured: false
publication: '*Mol Biol Evol*'
doi: 10.1093/molbev/msn123
---

Page content here