---
title: Selection Analysis Identifies Clusters of Unusual Mutational Changes in Omicron
  Lineage BA.1 That Likely Impact Spike Function
description:
url: ''
date: '2022-04-01'
publishDate: '2025-03-07T18:12:06.345780Z'
authors:
- Darren P Martin
- Spyros Lytras
- Alexander G Lucaci
- Wolfgang Maier
- Björn Grüning
- Stephen D Shank
- Steven Weaver
- Oscar A MacLean
- Richard J Orton
- Philippe Lemey
- Maciej F Boni
- Houriiyah Tegally
- Gordon W Harkins
- Cathrine Scheepers
- Jinal N Bhiman
- Josie Everatt
- Daniel G Amoako
- James Emmanuel San
- Jennifer Giandhari
- Alex Sigal
- ' NGS-SA'
- Carolyn Williamson
- Nei-Yuan Hsiao
- Anne von Gottberg
- Arne De Klerk
- Robert W Shafer
- David L Robertson
- Robert J Wilkinson
- B Trevor Sewell
- Richard Lessells
- Anton Nekrutenko
- Allison J Greaney
- Tyler N Starr
- Jesse D Bloom
- Ben Murrell
- Eduan Wilkinson
- Ravindra K Gupta
- Tulio de Oliveira
- Sergei L Kosakovsky Pond
publication_types:
- '2'
abstract: Among the 30 nonsynonymous nucleotide substitutions in the Omicron S-gene
  are 13 that have only rarely been seen in other SARS-CoV-2 sequences. These mutations
  cluster within three functionally important regions of the S-gene at sites that
  will likely impact (1) interactions between subunits of the Spike trimer and the
  predisposition of subunits to shift from down to up configurations, (2) interactions
  of Spike with ACE2 receptors, and (3) the priming of Spike for membrane fusion.
  We show here that, based on both the rarity of these 13 mutations in intrapatient
  sequencing reads and patterns of selection at the codon sites where the mutations
  occur in SARS-CoV-2 and related sarbecoviruses, prior to the emergence of Omicron
  the mutations would have been predicted to decrease the fitness of any virus within
  which they occurred. We further propose that the mutations in each of the three
  clusters therefore cooperatively interact to both mitigate their individual fitness
  costs, and, in combination with other mutations, adaptively alter the function of
  Spike. Given the evident epidemic growth advantages of Omicron overall previously
  known SARS-CoV-2 lineages, it is crucial to determine both how such complex and
  highly adaptive mutation constellations were assembled within the Omicron S-gene,
  and why, despite unprecedented global genomic surveillance efforts, the early stages
  of this assembly process went completely undetected.
featured: false
publication: '*Mol Biol Evol*'
tags:
- coevolution; epistasis; negative selection; positive selection
doi: 10.1093/molbev/msac061
---

Page content here