---
title: 'RASCL: Rapid Assessment of Selection in CLades through molecular sequence
  analysis'
description:
url: ''
date: '2022-01-01'
publishDate: '2025-03-07T18:12:05.079721Z'
authors:
- Alexander G Lucaci
- Jordan D Zehr
- Stephen D Shank
- Dave Bouvier
- Alexander Ostrovsky
- Han Mei
- Anton Nekrutenko
- Darren P Martin
- Sergei L Kosakovsky Pond
publication_types:
- '2'
abstract: 'An important unmet need revealed by the COVID-19 pandemic is the near-real-time
  identification of potentially fitness-altering mutations within rapidly growing
  SARS-CoV-2 lineages. Although powerful molecular sequence analysis methods are available
  to detect and characterize patterns of natural selection within modestly sized gene-sequence
  datasets, the computational complexity of these methods and their sensitivity to
  sequencing errors render them effectively inapplicable in large-scale genomic surveillance
  contexts. Motivated by the need to analyze new lineage evolution in near-real time
  using large numbers of genomes, we developed the Rapid Assessment of Selection within
  CLades (RASCL) pipeline. RASCL applies state of the art phylogenetic comparative
  methods to evaluate selective processes acting at individual codon sites and across
  whole genes. RASCL is scalable and produces automatically updated regular lineage-specific
  selection analysis reports: even for lineages that include tens or hundreds of thousands
  of sampled genome sequences. Key to this performance is (i) generation of automatically
  subsampled high quality datasets of gene/ORF sequences drawn from a selected \"query\"
  viral lineage; (ii) contextualization of these query sequences in codon alignments
  that include high-quality \"background\" sequences representative of global SARS-CoV-2
  diversity; and (iii) the extensive parallelization of a suite of computationally
  intensive selection analysis tests. Within hours of being deployed to analyze a
  novel rapidly growing lineage of interest, RASCL will begin yielding JavaScript
  Object Notation (JSON)-formatted reports that can be either imported into third-party
  analysis software or explored in standard web-browsers using the premade RASCL interactive
  data visualization dashboard. By enabling the rapid detection of genome sites evolving
  under different selective regimes, RASCL is well-suited for near-real-time monitoring
  of the population-level selective processes that will likely underlie the emergence
  of future variants of concern in measurably evolving pathogens with extensive genomic
  surveillance.'
featured: false
publication: '*PLoS One*'
doi: 10.1371/journal.pone.0275623
---

Page content here