---
title: Mapping protease inhibitor resistance to human immunodeficiency virus type
  1 sequence polymorphisms within patients
description:
url: ''
date: '2007-12-01'
publishDate: '2025-03-07T18:12:19.505323Z'
authors:
- Art F Y Poon
- Sergei L Kosakovsky Pond
- Douglas D Richman
- Simon D W Frost
publication_types:
- '2'
abstract: Resistance genotyping provides an important resource for the clinical management
  of patients infected with human immunodeficiency virus type 1 (HIV-1). However,
  resistance to protease (PR) inhibitors (PIs) is a complex phenotype shaped by interactions
  among nearly half of the residues in HIV-1 PR. Previous studies of the genetic basis
  of PI resistance focused on fixed substitutions among populations of HIV-1, i.e.,
  host-specific adaptations. Consequently, they are susceptible to a high false discovery
  rate due to founder effects. Here, we employ sequencing \"mixtures\" (i.e., ambiguous
  base calls) as a site-specific marker of genetic variation within patients that
  is independent of the phylogeny. We demonstrate that the transient response to selection
  by PIs is manifested as an excess of nonsynonymous mixtures. Using a sample of 5,651
  PR sequences isolated from both PI-naive and -treated patients, we analyze the joint
  distribution of mixtures and eight PIs as a Bayesian network, which distinguishes
  residue-residue interactions from direct associations with PIs. We find that selection
  for resistance is associated with the emergence of nonsynonymous mixtures in two
  distinct groups of codon sites clustered along the substrate cleft and distal regions
  of PR, respectively. Within-patient evolution at several positions is independent
  of PIs, including those formerly postulated to be involved in resistance. These
  positions are under strong positive selection in the PI-naive patient population,
  implying that other factors can produce spurious associations with resistance, e.g.,
  mutational escape from the immune response.
featured: false
publication: '*J Virol*'
doi: 10.1128/JVI.01570-07
---

Page content here