---
title: Phylogenetic analysis of population-based and deep sequencing data to identify
  coevolving sites in the nef gene of HIV-1
description:
url: ''
date: '2010-04-01'
publishDate: '2025-03-07T18:12:17.710705Z'
authors:
- Art F Y Poon
- Luke C Swenson
- Winnie W Y Dong
- Wenjie Deng
- Sergei L Kosakovsky Pond
- Zabrina L Brumme
- James I Mullins
- Douglas D Richman
- P Richard Harrigan
- Simon D W Frost
publication_types:
- '2'
abstract: Rapidly evolving viruses such as HIV-1 display extensive sequence variation
  in response to host-specific selection, while simultaneously maintaining functions
  that are critical to replication and infectivity. This apparent conflict between
  diversifying and purifying selection may be resolved by an abundance of epistatic
  interactions such that the same functional requirements can be met by highly divergent
  sequences. We investigate this hypothesis by conducting an extensive characterization
  of sequence variation in the HIV-1 nef gene that encodes a highly variable multifunctional
  protein. Population-based sequences were obtained from 686 patients enrolled in
  the HOMER cohort in British Columbia, Canada, from which the distribution of nonsynonymous
  substitutions in the phylogeny was reconstructed by maximum likelihood. We used
  a phylogenetic comparative method on these data to identify putative epistatic interactions
  between residues. Two interactions (Y120/Q125 and N157/S169) were chosen to further
  investigate within-host evolution using HIV-1 RNA extractions from plasma samples
  from eight patients. Clonal sequencing confirmed strong linkage between polymorphisms
  at these sites in every case. We used massively parallel pyrosequencing (MPP) to
  reconstruct within-host evolution in these patients. Experimental error associated
  with MPP was quantified by performing replicates at two different stages of the
  protocol, which were pooled prior to analysis to reduce this source of variation.
  Phylogenetic reconstruction from these data revealed correlated substitutions at
  Y120/Q125 or N157/S169 repeated across multiple lineages in every host, indicating
  convergent within-host evolution shaped by epistatic interactions.
featured: false
publication: '*Mol Biol Evol*'
doi: 10.1093/molbev/msp289
---

Page content here