---
title: 'A counting renaissance: combining stochastic mapping and empirical Bayes to
  quickly detect amino acid sites under positive selection'
description:
url: ''
date: '2012-12-01'
publishDate: '2025-03-07T18:12:14.925510Z'
authors:
- Philippe Lemey
- Vladimir N Minin
- Filip Bielejec
- Sergei L Kosakovsky Pond
- Marc A Suchard
publication_types:
- '2'
abstract: 'MOTIVATION: Statistical methods for comparing relative rates of synonymous
  and non-synonymous substitutions maintain a central role in detecting positive selection.
  To identify selection, researchers often estimate the ratio of these relative rates
  (dN/dS) at individual alignment sites. Fitting a codon substitution model that captures
  heterogeneity in dN/dS across sites provides a reliable way to perform such estimation,
  but it remains computationally prohibitive for massive datasets. By using crude
  estimates of the numbers of synonymous and non-synonymous substitutions at each
  site, counting approaches scale well to large datasets, but they fail to account
  for ancestral state reconstruction uncertainty and to provide site-specific dN/dS
  estimates. RESULTS: We propose a hybrid solution that borrows the computational
  strength of counting methods, but augments these methods with empirical Bayes modeling
  to produce a relatively fast and reliable method capable of estimating site-specific
  dN/dS values in large datasets. Importantly, our hybrid approach, set in a Bayesian
  framework, integrates over the posterior distribution of phylogenies and ancestral
  reconstructions to quantify uncertainty about site-specific dN/dS estimates. Simulations
  demonstrate that this method competes well with more-principled statistical procedures
  and, in some cases, even outperforms them. We illustrate the utility of our method
  using human immunodeficiency virus, feline panleukopenia and canine parvovirus evolution
  examples.'
featured: false
publication: '*Bioinformatics*'
doi: 10.1093/bioinformatics/bts580
---

Page content here